build.zig

```diff
@@ -287,6 +287,20 @@ fn getProtocInstallDir(
     allocator: std.mem.Allocator,
     protoc_version: []const u8,
 ) ![]const u8 {
+    if (std.process.getEnvVarOwned(allocator, "PROTOC_PATH")) |protoc_path| {
+        std.log.info("zig-protobuf: respecting PROTOC_PATH: {s}\n", .{ protoc_path });
+        if (fileExists(protoc_path)) {
+            // user has probably provided full path to protoc binary instead of proto_dir
+            // also, if these fail and user explicitly provided custom path, we probably don't want to download stuff
+            const bin_dir = std.fs.path.dirname(protoc_path).?;
+            const real_proto_dir = std.fs.path.dirname(bin_dir).?;
+            return real_proto_dir;
+        }
+
+        std.log.err("zig-protobuf: cannot resolve a protoc provided via PROTOC_PATH env var ({s}), make sure the value is correct", .{protoc_path});
+        std.process.exit(1);
+    } else |_| {}
```

Comment:
You can use `catch null` to avoid an empty error capture into an empty block in else.

```suggestion
    if (std.process.getEnvVarOwned(allocator, "PROTOC_PATH") catch null) |protoc_path| {
        std.log.info("zig-protobuf: respecting PROTOC_PATH: {s}\n", .{ protoc_path });
        if (fileExists(protoc_path)) {
            // user has probably provided full path to protoc binary instead of proto_dir
            // also, if these fail and user explicitly provided custom path, we probably don't want to download stuff
            const bin_dir = std.fs.path.dirname(protoc_path).?;
            const real_proto_dir = std.fs.path.dirname(bin_dir).?;
            return real_proto_dir;
        }

        std.log.err("zig-protobuf: cannot resolve a protoc provided via PROTOC_PATH env var ({s}), make sure the value is correct", .{protoc_path});
        std.process.exit(1);
    }
```